lib/import/import_sdf.rb

```diff
@@ -2,9 +2,9 @@
 
```

Comment:
General observation:

- It would make the file much more readable if the large context blocks e.g. 142 - 240 would be splitted into separate methods with self explaining names
- opening a transaction for every single sample seems to be a heavy load for the DB. Maybe separate the extracting part from the file to a intermediate format, then have a validation on that format and at the end only import the valid data in one transaction.